schemas/eq_submission_schema.json

```diff
@@ -197,22 +197,27 @@
     "case_type": {
       "$id": "#/properties/case_type",
       "type": "string",
+      "items": {
+        "type": "array",
+        "enum": ["HH", "HI", "CE", "CI"]
+      },
       "title": "The Case_type Schema",
       "default": "",
       "examples": [
         "HI"
       ],
-      "pattern": "^(.*)$"
+      "pattern": "^[A-Z]{2}$"
```

Comment:
This regex isn't necessary, since the values are now enumerated.

Reply:
Can you check these fixes I've made please?